Example/Podfile

```diff
@@ -4,4 +4,5 @@ use_frameworks!
 link_with 'AsakusaSatelliteSwiftClientExample', 'AsakusaSatelliteSwiftClientExampleTests'
 
 pod 'AsakusaSatellite', :path => '../'
+pod 'SwiftyJSON', :git => 'https://github.com/SwiftyJSON/SwiftyJSON.git', :branch => 'xcode6.3'
```

Comment:
this line should be removed before pod lib lint. awaiting SwiftyJSON spec for Swift 1.2.